models/Element/DirtyIndicatorInterface.php

```diff
@@ -15,7 +15,7 @@
  * @license    http://www.pimcore.org/license     GPLv3 and PEL
  */
 
-namespace Pimcore\Model\DataObject;
```

Comment:
you moved the file but forget to add the BC file in its original location, didn't you?

Reply:
Added. Thanks for review!